packages/roosterjs-editor-api/lib/format/setBackgroundColor.ts

```diff
@@ -1,6 +1,11 @@
 import execFormatWithUndo from './execFormatWithUndo';
 import { Editor } from 'roosterjs-editor-core';
 
+/**
+ * Set background color at current selection
```

Comment:
The comment is not accurate in no selection case.

Reply:
Current selection can be collapsed as well, so it can describe both cases. Or I can say:
'Set background color at current selection/cursor position.' Do you have any suggestions?